### DISCUSSION TITLE
Where is "Move file to another folder" command ?

### DISCUSSION THREAD

**Discussion by Fred-Vatin** [tadashi-aikawa/obsidian-another-quick-switcher#239] [Q&A] [answered]
Hello,

It stands in the doc that you can do that. But I can not see any command to display the `move dialog` in the `command palette` (only the built-in `move current file to another folder` is in the command palette):

<img src="https://github.com/tadashi-aikawa/obsidian-another-quick-switcher/assets/22337329/3bca9aa7-85f2-4c45-92b7-40716a0a21e2" width="500">

Or in the `main dialog`:

<img src="https://github.com/tadashi-aikawa/obsidian-another-quick-switcher/assets/22337329/ff0efc29-7827-43fb-9e3c-38bb2c0905d1" width="500">

It is only listed in hotkeys settings. Could you add this to command palette ?

**Comment by tadashi-aikawa** [comment]
Hi, @Fred-Vatin :)

> It stands in the doc that you can do that. But I can not see any command to display the `move dialog` in the `command palette` (only the built-in `move current file to another folder` is in the command palette):

Unfortunately, I couldn't reproduce it. Could you tell me the reproduction steps in the sandbox vault?

![image](https://github.com/tadashi-aikawa/obsidian-another-quick-switcher/assets/9500018/ce711c08-4362-4e78-a0d8-0494b39786a2)

**Comment by Fred-Vatin** [comment, marked as answer]
Don’t bother. I’m stupid. I had hidden the command some days ago in **Better command palette**.
